astropy/io/ascii/tests/test_ecsv.py

```diff
@@ -267,15 +267,10 @@ def assert_objects_equal(obj1, obj2, attrs, compare_class=True):
     if compare_class:
         assert obj1.__class__ is obj2.__class__
 
-    # For a column that is a native astropy Column, ignore the specified
```

Comment:
Note: this removal is not strictly necessary, but initially I added 'dtype' to the list of attributes to be explicitly tested, and was very confused that did not work. I think it is better that it is explained inside a test why something specific is not checked, rather than here. Obviously can revert if needed.